api4/system.go

```diff
@@ -7,15 +7,30 @@ import (
 	"net/http"
 
 	l4g "github.com/alecthomas/log4go"
+	"github.com/mattermost/platform/app"
+	"github.com/mattermost/platform/model"
 	"github.com/mattermost/platform/utils"
 )
 
 func InitSystem() {
 	l4g.Debug(utils.T("api.system.init.debug"))
 
 	BaseRoutes.System.Handle("/ping", ApiHandler(getSystemPing)).Methods("GET")
+	BaseRoutes.System.Handle("/config", ApiSessionRequired(getConfig)).Methods("GET")
```

Comment:
Can we just make this `/config` without the system prefix? It can stay in this file but just change `BaseRoutes.System` to `BaseRoutes.ApiRoot`